src/__tests__/Link-spec.jsx

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
It might be worth writing a test to make sure it's outputting an anchor tag. Maybe we could use snapshots for this.

Reply:
The render function just returns an <a> with given props and children. Do we really need to test that?

Reply:
added snapshot!